packages/angular/cli/BUILD.bazel

```diff
@@ -316,7 +316,14 @@ genrule(
 
 pkg_npm(
     name = "npm_package",
-    srcs = [":package.json"],
+    pkg_deps = [
+        "//packages/angular_devkit/architect:package.json",
+        "//packages/angular_devkit/build_angular:package.json",
+        "//packages/angular_devkit/build_webpack:package.json",
+        "//packages/angular_devkit/core:package.json",
+        "//packages/angular_devkit/schematics:package.json",
+        "//packages/schematics/angular:package.json",
+    ],
```

Comment:
When doing tar substitution we need the dependent package.json files so that we can get the package name and change:
```
  "dependencies": {
    "@some-package: "0.0.0",
  }
```
to
```
  "dependencies": {
    "@some-package: "file:/path/to/pacakge.tar.gz",
  }
```

Reply:
Unlike release and snapshot stamping, which just needs to look for `0.0.0` and replace that.